lib/README.md

```diff
@@ -0,0 +1,8 @@
+#### Running the program in IRB
```

Comment:
It'd be great to include a sample usage of your code in the readme. This would help an end-user see how they should use the classes and methods you have written.